developer/src/kmc-generate/src/template/wordlist-lexical-model/source/wordlist.tsv

```diff
@@ -0,0 +1,16 @@
+#
+# $NAME $VERSION generated from template.
+#
+# This is an example tab-separated wordlist file that can be edited in a spreadsheet
+# program or regenerated from a wordlist tool. See lexical model documentation at 
+# https://help.keyman.com/developer/ for tools.
+#
+# Columns
+# =======
+# WORD	FREQUENCY	(Optional)NOTES
+#
+the	100
+example	5
+hello	10
+world	8
+wordlist	3
```

Comment:
I believe these are tabs in the actual file.
On the GH PR view, they shows as spaces. :+1

Reply:
Yes, they are tabs.